fakeprod/datarepo-api.yaml

```diff
@@ -9,6 +9,13 @@ env:
   DB_STAIRWAY_USERNAME: drmanager
   DB_STAIRWAY_URI: jdbc:postgresql://terra-jade-gcloud-sqlproxy.data-repo:5432/stairway
   DB_DATAREPO_URI: jdbc:postgresql://terra-jade-gcloud-sqlproxy.data-repo:5432/datarepo
+  DATAREPO_RESOURCEID: datarepo-terra
+  DATAREPO_DNSNAME: jade-terra.datarepo-prod.broadinstitute.org
+  DATAREPO_GCS_BUCKET: broad-datarepo-terra-prod
```

Comment:
Does prod need the bucket? I think we are using real projects and creating new buckets so we should not need that.